### PR TITLE
Using layers

### PR DESCRIPTION
**Pull request by svenngjeto**
Source code is same as before, code is moved to sub-projects -> templates and build.gradleis updated accordingly

### CODE REVIEW

File: search-index-import-util/src/main/java/no/unit/nva/dynamodb/DynamoDBExportFileReader.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    private final AmazonS3 s3Client;
+
+    public DynamoDBExportFileReader(ElasticSearchHighLevelRestClient elasticSearchRestClient, AmazonS3 s3Client) {
+        this.elasticSearchRestClient = elasticSearchRestClient;
+        this.s3Client = s3Client;
+    }
+
+    private static boolean isPublished(JsonNode jsonNode) {
+        var statusNode = jsonNode.get(STATUS);
+        return !isNull(statusNode)
```
Comment (brinxmat) [inline search-index-import-util/src/main/java/no/unit/nva/dynamodb/DynamoDBExportFileReader.java:52]: !isNull?

File: search-index-import-util/src/test/resources/datapipeline_output_sample [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{"owner":{"s":"sase@unit.no"},"identifier":{"s":"9b98c263-6974-49bc-b42b-2fa9562c6432"},"publisherId":{"s":"https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934"},"createdDate":{"s":"2020-09-28T11:57:31.466056Z"},"publisherOwnerDate":{"s":"https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934#sase@unit.no#2020-09-28T11:57:31.466056Z"},"modifiedDate":{"s":"2020-09-28T11:57:31.466056Z"},"publisher":{"m":{"id":{"s":"https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934"},"type":{"s":"Organization"}}},"type":{"s":"Publication"},"status":{"s":"Draft"},"fileSet":{"m":{"files":{"l":[{"m":{"identifier":{"s":"e0cf7d7a-f636-4bc0-ac9c-fcbdddfd4c8f"},"size":{"n":"164161"},"publisherAuthority":{"bOOL":false},"name":{"s":"Skjermbilde 2020-09-25 kl. 11.28.22.png"},"administrativeAgreement":{"bOOL":false},"mimeType":{"s":"image/png"},"type":{"s":"File"}}}]},"type":{"s":"FileSet"}}}}
```
Comment (brinxmat) [inline search-index-import-util/src/test/resources/datapipeline_output_sample:1]: Should this file have extension ".json"?

File: search-index-import-util/src/main/java/no/unit/nva/dynamodb/DynamoDBExportFileReader.java (hunk 1 above)
Comment (svenngjeto) [reply search-index-import-util/src/main/java/no/unit/nva/dynamodb/DynamoDBExportFileReader.java]: Turned double negation into something positive

File: search-index-import-util/src/test/resources/datapipeline_output_sample (hunk 2 above)
Comment (svenngjeto) [reply search-index-import-util/src/test/resources/datapipeline_output_sample:1]: Maybe - It's not valid json, it is just json stacked in a file  (thanks to DataPipeline)

**Review by brinxmat — APPROVED** [review]

**Pull request merged**